index.html

```diff
@@ -27,39 +27,110 @@
     </div>
     <div class="intro-text">
       <h1 class="text-center nameplate">Jonathan Chickie</h1>
-      <hr class="star-light"></hr>
-      <h3 class="text-center subtitle">Web Developer - Graphic Artist - User Experience Designer</h3>
+      <hr class="bird-light"></hr>
+      <h3 class="text-center subtitle"><i class="fa fa-globe"></i>Web Developer<i class="fa fa-paint-brush"></i>Graphic Artist<i class="fa fa-puzzle-piece"></i>User Experience Designer</h3>
     </div>
   </div>
 
-  <div class="portfolio">
-    <div class="container">
-      <h1 class="text-center section-header">Portfolio</h1>
-      <hr class="star-dark"></hr>
-    </div>
-    <div class="container-fluid">
-      <div class="row">
-        <div class="col-md-2 well well-lg col-md-offset-2"></div>
-        <div class="col-md-2 well well-lg col-md-offset-1"></div>
-        <div class="col-md-2 well well-lg col-md-offset-1"></div>
+  <div class="portfolio container">
+    <h1 class="text-center section-header">Portfolio</h1>
```

Comment:
Good to know! I've used CSS to center things in the past, but was taking advantage of the Bootstrap class.